integration/integration_test.go

```diff
@@ -1281,25 +1281,26 @@ func testEscapeSequenceNoTrigger(t *testing.T, terminal *Terminal, sess <-chan e
 
 type localAddr struct {
 	mu   sync.Mutex
-	addr net.Addr
+	addr map[string]struct{}
```

Comment:
Rename to plural and document key?

```suggestion
	addrs map[string]struct{} // key is the string representation of a net.Addr, eg: "localhost:8080"
```

(Correct it wrong, ofc.)